client/src/componets/BlogDetail.js

```diff
@@ -34,7 +34,7 @@ const BlogDetail = () => {
         description: data.blog.description,
       });
     });
-  }, [id]);
+  });
```

Comment:
Better to add `[]` as the dependency to avoid possibility of infinite rendering.

Reply:
I have remove the dependency array [ ] as it is showing a warning on  the console 

"WARNING in [eslint] 
src/componets/BlogDetail.js
  Line 37:6:  React Hook useEffect has a missing dependency: 'fetchDetails'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

webpack compiled with 1 warning"

but when I remove it warning is gone.